File: TestSample/csharp/ListView_Performance_Test/WPF_ListView/DataModel.cs
using System.ComponentModel;
using System.Runtime.CompilerServices;

namespace Performance_Test
{
    public class DataModel : INotifyPropertyChanged
    {
        private int _userId;
        private int _id;
        private string _title = string.Empty;
        private string _body = string.Empty;

        public int UserId
        {
            get => _userId;
            set
            {
                _userId = value;
                OnPropertyChanged();
            }
        }

        public int Id
        {
            get => _id;
            set
            {
                _id = value;
                OnPropertyChanged();
            }
        }

        public string Title
        {
            get => _title;
            set
            {
                _title = value;
                OnPropertyChanged();
            }
        }

        public string Body
        {
            get => _body;
            set
            {
                _body = value;
                OnPropertyChanged();
            }
        }

        #region INotifyPropertyChange Implementation
        public event PropertyChangedEventHandler? PropertyChanged = delegate { };
        protected void OnPropertyChanged([CallerMemberName] string propertyName = null)
        {
            PropertyChangedEventHandler? handler = this.PropertyChanged;

            if (handler != null)
            {
                handler(this, new PropertyChangedEventArgs(propertyName));
            }
        }
        #endregion INotifyPropertyChange Implementation
    }
}

File: TestSample/csharp/ListView_Performance_Test/Flutter_ListView/lib/data_model.dart
class DataModel {
  final int userId;
  final int id;
  final String title;
  final String body;

  DataModel(
      {required this.userId,
      required this.id,
      required this.title,
      required this.body});

  factory DataModel.fromJson(Map<String, dynamic> json) => DataModel(
      userId: json['userId'] as int,
      id: json['id'] as int,
      title: json['title'],
      body: json['body']);
}
